app/assets/javascripts/mtl/submit.coffee

```diff
@@ -0,0 +1,3 @@
+$(document).on 'click', '[data-mtl-submit]', (ev) ->
```

Comment:
Ao we require a preventDefault() here maybe, so when it's e.g. bound to a `<a>` tag?